### PR TITLE
Hog fixes 3.0

### PR DESCRIPTION
**Pull request by Carbonhell**
Datumizes sides and ranks along with fixing hud problem on aghost/relogging & fixing round end message. Also a few other little stuff.

### CODE REVIEW

**Comment by Nokington** [comment]
aeyh suer

**Comment by Carbonhell** [comment]
Rechecked code, it seems okay

File: code/datums/hud.dm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /* HUD DATUMS */
 
 //GLOBAL HUD LIST
-var/datum/atom_hud/huds = list( \
+var/list/datum/atom_hud/huds = list( \
```
Comment (Carbonhell) [inline code/datums/hud.dm:4]: While coding,i've seen that this list isn't declared as a list. That made list procs not work such as list.Add(stuff) and other things,so i've added it, just to be clear

**Comment by Carbonhell** [comment]
Wait,found an issue with godspeak ability

**Comment by FuryMcFlurry** [comment]
uh oh.
that doesn't sound good.

**Comment by Carbonhell** [comment]
Ok i just found a few serious bugs,do not merge for now

**Comment by Carbonhell** [comment]
Should be good to go now.

**Comment by Carbonhell** [comment]
One bug left,will fix today,dnm

**Comment by Carbonhell** [comment]
Bug fixed, good to merge

**Comment by nalarmg** [comment]
Needs code review by another coder.

**Comment by Nebyab** [comment]
sure

**Comment by Carbonhell** [comment]
Checked the code all over again, it should be 100% working now. is this enough...?

**Comment by SplergeBucket** [comment]
This code is okay.
Paid endorsement by Carbonhell in exchange for five goodboy points, I expect to see it in my mail.

**Comment by Carbonhell** [comment]
2 headmi ok and one code revision, merge when

**Pull request merged**